shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.js

```diff
@@ -348,9 +180,182 @@ exports.serveCaseToIrsInteractor = async (
       }));
   }
 
+  return urlToReturn;
+};
+
+const createCoversheetsForServedEntries = async ({
+  applicationContext,
+  caseEntity,
+}) => {
+  for (const doc of caseEntity.docketEntries) {
+    if (doc.isFileAttached) {
+      const updatedDocketEntry = await applicationContext
+        .getUseCases()
+        .addCoversheetInteractor(applicationContext, {
+          caseEntity,
+          docketEntryId: doc.docketEntryId,
+          docketNumber: caseEntity.docketNumber,
+          replaceCoversheet: !caseEntity.isPaper,
+          useInitialData: !caseEntity.isPaper,
+        });
+
+      caseEntity.updateDocketEntry(updatedDocketEntry);
+    }
+  }
+};
+
+const generatePaperNoticeForContactSecondary = async ({
+  applicationContext,
+  caseCaptionExtension,
+  caseEntity,
+  caseTitle,
+  contactSecondary,
+  docketNumberWithSuffix,
+  pdfData,
+  preferredTrialCity,
+  receivedAt,
+}) => {
+  const contactInformationDiff = applicationContext
+    .getUtilities()
+    .getAddressPhoneDiff({
+      newData: caseEntity.petitioners[0],
+      oldData: contactSecondary,
+    });
+
+  const addressFields = [
+    'country',
+    'countryType',
+    'address1',
+    'address2',
+    'address3',
+    'city',
+    'state',
+    'postalCode',
+  ];
+
+  const contactAddressesAreDifferent = Object.keys(contactInformationDiff).some(
+    field => addressFields.includes(field),
+  );
+
+  let secondaryPdfData;
+
+  if (contactAddressesAreDifferent) {
+    secondaryPdfData = await applicationContext
```

Comment:
nit, but we could just `return;` if they're the same, and then continue execution with a const after without having to include this in a block.

```
if (!contactAddressesAreDifferent) {
  return;
}

const secondaryPdfData = = await applicationContext...
```

Reply:
addressed

Reply:
not seeing it yet...